terraform/digitalgov.gov.tf

```diff
@@ -122,46 +122,35 @@ resource "aws_route53_record" "find_digitalgov_gov_a" {
   ]
 }
 
-# END REDIRECTS
-
-# =================================
-
-# SERVICES
+# U.S. Digital Registry
+# Redirects to cloud.gov External Domains using https://github.com/18f/pages-redirects
+# Contact feedback-analytics@gsa.gov regarding changes
 
 # U.S. Digital Registry
 # usdigitalregistry.digitalgov.gov
-resource "aws_route53_record" "usdigitalregistry_digitalgov_gov_a" {
+resource "aws_route53_record" "redirect_usdigitalregistry_digitalgov_gov" {
   zone_id = aws_route53_zone.digitalgov_gov_zone.zone_id
   name    = "usdigitalregistry.digitalgov.gov."
   type    = "CNAME"
-  ttl     = "300"
-  records = [
-    "alb-smr-prod-pub-2-970074805.us-east-1.elb.amazonaws.com."
-  ]
+  ttl     = 300
+  records = ["usdigitalregistry.digitalgov.gov.external-domains-production.cloud.gov."]
 }
 
-# U.S. Digital Registry / Staging
-# stage-socialmobileregistry.digitalgov.gov
-resource "aws_route53_record" "stage-socialmobileregistry_digitalgov_gov_a" {
+# U.S. Digital Registry (www)
+# www.usdigitalregistry.digitalgov.gov
+resource "aws_route53_record" "redirect_www_usdigitalregistry_digitalgov_gov" {
```

Comment:
By changing the resource name this is going to create a new record not update the existing resource. It would be cleaner to keep the resource name/id and update the records

Reply:
this one is different and not staging, so, i'm inclined to keep this